src/components/SearchBar/index.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { withWindow } from 'react-window-decorators';
+import PropTypes from 'prop-types';
+
+import Input from '../Input';
+import Checkbox from '../Checkbox';
+import Button from '../Button';
+
+import './SearchBar.scss';
+
+const SearchBar = ({ ...props }) => {
+  const checkboxLabel = () => {
+    return props.breakpoint === 'large' ? 'Full Time Only' : 'Full Time';
```

Comment:
Try to avoid using strings as values to check. When you define names as an object, you should use it all over the project like
```
... props.breakpoint === names.LARGE ? ....
```